node-fs-writefile/cp.js

```diff
@@ -0,0 +1,9 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+try {
+  const file = await readFile(process.argv[2]);
+  const write = process.argv[3];
+  await writeFile(write, `${file} \n`);
```

Comment:
By using string interpolation you are forcing the written file to be text. That's OK, but it means this code won't copy binary files (like images). To get it to work for binary files, just pass `file` directly to `writeFile`!.